app/controllers/concerns/curation_concerns/collections_controller_behavior.rb

```diff
@@ -15,6 +15,9 @@ module CollectionsControllerBehavior
 
       # This is needed as of BL 3.7
       copy_blacklight_config_from(::CatalogController)
+      configure_blacklight do |config|
+        config.search_builder_class = CurationConcerns::CollectionSearchBuilder
```

Comment:
I'm pretty sure this isn't necessary because `#index` is calling `collection_search_builder` directly.